display/d.graph/do_graph.c

```diff
@@ -384,14 +389,15 @@ int do_symbol(const char *str)
         D_use_color(D_parse_color(DEFAULT_BG_COLOR, 0));
     else /* unset or bad */
         D_RGB_color(line_color->r, line_color->g, line_color->b);
-
+cleanup_and_exit:
     G_free(symb_name);
     G_free(line_color_str);
     G_free(fill_color_str);
     G_free(line_color);
     G_free(fill_color);
+    G_free(Symb);
```

Comment:
A `SYMBOL` is a struct, which can't be simply freed. Please take a look att its definition.